src/exporter.h

```diff
@@ -0,0 +1,193 @@
+#ifndef EXPORTER_H_
+#define EXPORTER_H_
+
+#include <array>
+#include <cmath>
+#include <fstream>
+#include <iostream>
+#include <sstream>
+#include <string>
+
+#include "backend.h"
+#include "daosoa.h"
+#include "default_force.h"
+#include "inline_vector.h"
+#include "math_util.h"
+#include "param.h"
+
+using std::cout;
+using std::endl;
+using std::string;
+using std::fstream;
+using std::ofstream;
+
+namespace bdm {
+
+class Exporter {
+ public:
+  /// This function exports the cell positions into a file,
+  /// where each line contains the 3D position of a cell in square brackets.
+  template <typename daosoa>
+  void ToFile(daosoa* cells, string filename) const {
```

Comment:
I would recommend making the function definition `const - correct`
`cells` are not modified inside `ToFile`. I would therefore recommend to pass it as const-reference
`void ToFile(const daosoa& cells, string filename) const {`
`const-correctness` is another type safety measure. The compiler would throw an error if cells would be mutated. As a result. the caller of this function can be sure that `cells` won't be modified inside this function. 
I have overlooked that during the last reviews

-> inside the function all occurences of `(*cells)` must be replaced with `cells`
-> a call to this function `export.ToFile(&cells);` must be changed to `export.ToFile(cells)`